src/main/java/cs2103/v15_1j/jimjim/model/TaskEvent.java

```diff
@@ -21,5 +21,10 @@ public void setName(String name){
 	public StringProperty taskNameProperty() {
 		return name;
 	}
-
+	
+	@Override
+	public boolean equals(Object t) {
```

Comment:
this is not the right way to implement `equals` though. You need to check whether t is null, whether t is an instanceof TaskEvent etc.